tests/docs/listings/_quarto.yml

```diff
@@ -5,4 +5,4 @@ website:
   title: "Listing Test"
 format:
   html:
-    theme: [litera, styles.scss]
\ No newline at end of file
+    theme: [litera]
```

Comment:
I don't think this makes a difference either way, but why have this change here?

Reply:
To remove all the warnings from the log because the file was not found.
It makes it harder to focus on what matters when testing this.